### PR TITLE
doc: 1967 send otp

### PR DESCRIPTION
**Pull request by eoln**
describe how **/sendOTP** example endpoint (delivered by simulator) should be used during Linking OTP flow
https://app.zenhub.com/workspaces/mojaloop-project-59edee71d1407922110cf083/issues/mojaloop/project/1967

### CODE REVIEW

File: docs/linking/2-request-consent-otp.puml [hunk 1]
```diff
@@ -56,6 +56,12 @@ Switch -> DFSP ++:  ""POST /consentRequests""\n\
 DFSP --> Switch: ""202 Accepted""
 deactivate Switch
 
+DFSP -> DFSP : ""POST /sendOTP""\n\
```
Comment (kleyow) [inline docs/linking/2-request-consent-otp.puml:59]: DFSP to DFSP? just double checking that this is correct.

**Review by kleyow — APPROVED** [review]
+1. 1 question though.

File: docs/linking/2-request-consent-otp.puml (hunk 1 above)
Comment (eoln) [reply docs/linking/2-request-consent-otp.puml:59]: I didn't want to introduce a new actor. I want to only illustrate that DFSP is responsible to send SMS. This endpoint will be needed when we will implement linking flow in `thridparty-scheme-adapter`.

Comment (kleyow) [reply docs/linking/2-request-consent-otp.puml:59]: 👍

**Pull request merged**